client/js/widgets/widget.js

```diff
@@ -1300,6 +1300,52 @@ export class Widget extends StateManaged {
         }
       }
 
+      if(a.func == 'SCORE') {
+        setDefaults(a, { mode: 'set', property: 'score', seats: null, round: null, value: null});
+        if([ 'set', 'inc', 'dec' ].indexOf(a.mode) == -1) {
+          problems.push(`Warning: Mode ${a.mode} interpreted as set.`);
+          a.mode = 'set'
+        }
+        
+        if(a.value === null)
+          a.value = a.mode=='set' ? 0 : 1;
+        if(isNaN(parseFloat(a.value))) {
+          problems.push(`value ${a.value} must be a number, assuming 0.`);
+          a.value = 0;
+        }
+        a.value = parseFloat(a.value);
+
+        let round = a.round;
```

Comment:
While you are checking that round parses to an integer, it can still remain a string for the rest of this method. It would be better to force it to an integer here so that there's no chance of accidentally performing a string operation later, i.e.
```
let round = a.round ? parseInt(a.round) : null;
```

Reply:
Good catch. Done.